OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -121,10 +123,20 @@ public override Activity Tick(Actor self)
 				}
 
 				if (!onRailsHack)
-					attack.Target = Target.Invalid;
+					attack.Key.Target = Target.Invalid;
 
 				return NextActivity;
 			}
+
+			IEnumerable<KeyValuePair<AttackFollow, Armament>> SelectActiveAttack(Target t, bool forceAttack)
```

Comment:
I don't plan per se, but @pchote's longterm plan explained in https://github.com/OpenRA/OpenRA/pull/11600#discussion_r70411452 would involve the attack activity handling all the pairs returned.

However it's way out of scope for this PR IMO.